frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1329,6 +1329,7 @@ function ReaderHighlight:onHold(arg, ges)
     -- otherwise, we must be holding on text
     if self.view.highlight.disabled then return false end -- Long-press action "Do nothing" checked
     local ok, word = pcall(self.ui.document.getWordFromPosition, self.ui.document, self.hold_pos)
+    logger.dbg("LOGG-1 RH.lua onHold: word =", word, "ok =", ok)
```

Comment:
If this is meant for merging rather than as merely personal notes for debugging I'd write out ReaderHightlight in full so it's self-explanatory when looking at the logs. Also I suspect `RH` is more likely to yield false positives when grepping.

Reply:
This is just for personal notes hence draft.

Reply:
(I leave it as draft in case I actually find something useful, so I can modify this PR. It did strike me though that I should have probably left this inside my own fork - so, sorry for the noise :D).

Reply:
No problem. You can also do PRs within your own fork btw.